src/MCPClient/lib/clientScripts/policy_check.py

```diff
@@ -230,8 +232,8 @@ def _execute_rule_command(self, rule):
             output = json.loads(stdout)
         except ValueError:
             logger.exception(
-                'Unable to load an object from the malformed JSON in\n%s',
-                stdout)
+                'Unable to load an object from the malformed JSON: ',
+                stderr)
```

Comment:
This will probably raise an exception because the `%s` is now missing. I would have thought `PolicyChecker` had coverage, but it doesn't seem so.

Reply:
:woman_facepalming:

Reply:
Sorry! Dunno what I was thinking